### PR TITLE
feat(run-protocol): preview launch support, PSM work-around

### LINKED ISSUE #5010: Integrate PSM with bootstrap

### PR DESCRIPTION
**Pull request by dckc**
refs: #5062
closes: #5010
modulo #5218

## Description

This is the work necessary to get a clean preview deploy in **agoricstage-23**.

These are not work-arounds, so they should land:
 - cfa36cd2c feat(vats): export AssetTerms type from mintHolder.js
 - 423f67c76 refactor(run-protocol): move startPSM() to src/psm/startPSM.js
 - 0bce9e331 fix(run-protocol): PSM governance, anchor mintHolder
 - 9e7e729ad chore(run-protocol): write 2 proposals: econ-committee, main
 - 16546a92a docs(run-protocol): preview docs, WIP
 - 7dc988ff8 build(run-protocol): deploy-contracts script
 - bc7cd251a build(run-protocol): build:eval:main, deploy:contracts npm scripts
 - 07685b619 build(run-protocol): Makefile for agd tx gov submit-proposal etc.

These are the PSM work-around for version skew with **agoricdev-10**. Let's land these for the short term; #5218 represents work to clean them up:
 - 24f4c784e chore(run-protocol): exclude PSM_MANIFEST from MAIN for now (WIP)
 - 10fe1a771 build(run-protocol): writePsmScript for bootstrap version skew
 - 8ce23d048 build(run-protocol): PSM work-around npm scripts

This is a generated artifact; I kept it checked in so that I could track how it was changing.

 - a1cc82f2b build(run-protocol): generated gov-startPSM.js script

### CODE REVIEW

**Review by michaelfig — APPROVED** [review]
LGTM!  My only concerns are around what we need to do to reduce technical debt, but that can wait until next devnet.

File: packages/run-protocol/src/econ-behaviors.js [hunk 1]
```diff
@@ -74,7 +73,7 @@ export const startEconomicCommittee = async (
   },
   electorateTerms = {
     committeeName: 'Initial Economic Committee',
-    committeeSize: 1,
+    committeeSize: 3,
```
Comment (michaelfig) [inline packages/run-protocol/src/econ-behaviors.js:76]: This apparently causes a failure at https://github.com/Agoric/agoric-sdk/runs/6167083776?check_suite_focus=true#step:5:9111

File: packages/run-protocol/package.json [hunk 2]
```diff
@@ -10,7 +10,10 @@
   "scripts": {
     "build": "yarn build:bundles",
```
Comment (turadg) [inline packages/run-protocol/package.json:11]: Should this build * like 'lint' does?

**Review by turadg — APPROVED** [review]
run-protocol change LGTM

File: packages/run-protocol/package.json (hunk 2 above)
Comment (dckc) [reply packages/run-protocol/package.json:11]: perhaps... but some of the build:* things involve blockchain access, so they don't fit the same workflow.

**Pull request merged**